infrastructure/containers/postgres/automatic-backup/backup_or_restore.py

```diff
@@ -145,17 +259,19 @@ def restore_database(backup_fpath):
 
 
 def main():
+    print("System args:", sys.argv)
     print("Running database-backup process with subcommand", sys.argv[1])
     if sys.argv[1] == "backup":
-        backup_database(make_backup_fpath())
+        s3_backup_object_name = make_backup_object_name_from_datetime()
+        backup_database(s3_backup_object_name)
     elif sys.argv[1] == "backup-on-interval":
         backup_interval_seconds = parse_timedelta(BACKUP_INTERVAL).seconds
         backup_database_on_interval(seconds=backup_interval_seconds)
     elif sys.argv[1] == "restore-from-most-recent":
-        restore_database(get_most_recent_backup_fpath())
+        restore_database()
     elif sys.argv[1] == "restore-from-backup":
-        backup_fpath = BACKUP_DIR / sys.argv[2]
-        restore_database(backup_fpath)
+        s3_backup_object_name_to_restore_from__override = sys.argv[2]
```

Comment:
Another type annotation

```suggestion
        s3_backup_object_name_to_restore_from__override: str = sys.argv[2]
```